### ISSUE TITLE
Can't disable Style/FrozenStringLiteralComment in file with shebang

### ISSUE THREAD

**Issue by aroben** [bbatsov/rubocop#3722]
In a file with a shebang, there seems to be no way to add a `rubocop:disable` comment to disable the `Style/FrozenStringLiteralComment` cop.

Given this file `test.rb`:

```ruby
#!/usr/bin/env ruby

str = ""
str << "hello world"
puts str
```

Rubocop will complain about a missing frozen string literal comment, as expected:

```shellsession
$ rubocop test.rb
Inspecting 1 file
C

Offenses:

test.rb:1:1: C: Missing frozen string literal comment.
#!/usr/bin/env ruby
^

1 file inspected, 1 offense detected
```

As you can see, the file isn't compatible with frozen string literals, so I'd like to add a `rubocop:disable` comment to turn off that cop in this file. This works, but breaks the shebang line:

```ruby
# rubocop:disable Style/FrozenStringLiteralComment
#!/usr/bin/env ruby

str = ""
str << "hello world"
puts str
```

This does not work, but I wish it did:

```ruby
#!/usr/bin/env ruby
# rubocop:disable Style/FrozenStringLiteralComment

str = ""
str << "hello world"
puts str
```

## RuboCop version

```
$ bundle exec rubocop -V
0.45.0 (using Parser 2.3.1.4, running on ruby 2.2.0 x86_64-darwin15)
```

**Comment by tejasbubane** [comment]
You can also disable that cop using `frozen_string_literal: false` which works well with `shebang`:

This works:
```ruby
#!/usr/bin/env ruby
# frozen_string_literal: true

str = ""
str << "hello world"
puts str
```

Tested on:
```
$  rubocop -V
0.45.0 (using Parser 2.3.1.4, running on ruby 2.3.1 x86_64-darwin15)
```

There is a spec for this [here](https://github.com/bbatsov/rubocop/blob/master/spec/rubocop/cop/style/frozen_string_literal_comment_spec.rb#L64-L70).

Although, I think it would be better if `rubocop:disable` respects `shebang` in general.

**Comment by aroben** [comment]
Oh, that's great! Thanks for the tip!

> Although, I think it would be better if `rubocop:disable` respects `shebang` in general.

Agreed. I think this specific issue can be closed though.

**Comment by pboling** [comment]
It seems this has has not yet been fixed (preventing this cop from complaining about shebang Ruby files), and I just ran into it.

The fix suggested may seem like it has the effect of "disabling" the cop, but that's not really what is happening.  It is satisfying the cop by actually adding the Frozen String Literal comment it wants to see, except setting it to `false` instead of the more common `true`, which may not be desirable.

**Comment by dvandersluis** [comment]
@pboling can you give an example? `Style/FrozenStringLiteralComment` should be handling files with shebangs:

https://github.com/rubocop/rubocop/blob/afe639e359b3360cde162e249618f91fe6a2903f/spec/rubocop/cop/style/frozen_string_literal_comment_spec.rb#L49-L61

**Comment by pboling** [comment]
Apologies, when I run the cop explicitly with `--only`, it does work.  I was mainly just pointing out that `# frozen_string_literal: false` does not *disable* the cop, it *satisfies* the cop.
